concepticondata/conceptlists/Desrochers-2010-330.tsv

```diff
@@ -0,0 +1,331 @@
+Desrochers-2010-330-267	267	fustigar	whip			Verb	2.77450980392157	8	5	
+Desrochers-2010-330-268	268	garabatear	scribble			Verb	3.38235294117647	10		
+Desrochers-2010-330-269	269	guardar	protect	PROTECT	2900	Verb	6.2843137254902	7	116	6.76
+Desrochers-2010-330-270	270	hacinar	pile up			Verb	2.31372549019608	7		
+Desrochers-2010-330-271	271	haraganear	lounge			Verb	1.8921568627451	10		
+Desrochers-2010-330-272	272	holgar	be superfluous			Verb	2.76470588235294	6	3	
+Desrochers-2010-330-273	273	hurtar	steal	STEAL	713	Verb	3.57430143213565	6	6	
+Desrochers-2010-330-274	274	identificar	identify			Verb	6.12	11	79	5.25
+Desrochers-2010-330-275	275	imbuir	imbue			Verb	1.70588235294118	6	1	
+Desrochers-2010-330-276	276	implorar	begging			Verb	3.00980392156863	8	5	
+Desrochers-2010-330-277	277	invocar	invoke			Verb	3.96078431372549	7	20	3.65
+Desrochers-2010-330-278	278	jalar	pull	PULL	1455	Verb	4.09803921568627	5	2	
+Desrochers-2010-330-279	279	jamar	polish off			Verb	2.31372549019608	5		
+Desrochers-2010-330-280	280	lamer	lick	LICK	319	Verb	4.66666666666667	5	6	
+Desrochers-2010-330-281	281	laminar	laminar			Verb	2.72549019607843	7	4	
+Desrochers-2010-330-282	282	lanzar	launch			Verb	5.50980392156863	6	93	5.28
+Desrochers-2010-330-283	283	lavar	wash	WASH	1453	Verb	6.41176470588235	5	35	6.35
+Desrochers-2010-330-284	284	limpiar	clean	CLEAN (SOMETHING)	2235	Verb	6.63725490196079	7	57	6.49
+Desrochers-2010-330-285	285	lustrar	polish	POLISH (SOMETHING)	3659	Verb	2.42156862745098	7	6	
+Desrochers-2010-330-286	286	marcar	dial			Verb	5.87254901960784	6	69	6.03
+Desrochers-2010-330-287	287	matar	kill	KILL	1417	Verb	5.92156862745098	5	239	5.52
+Desrochers-2010-330-288	288	meter	put	PUT	998	Verb	6.4159747889007	5	130	6.68
+Desrochers-2010-330-289	289	mezclar	mix	MIX	2017	Verb	5.82352941176471	7	24	6.12
+Desrochers-2010-330-290	290	mitigar	mitigate			Verb	2.95098039215686	7	6	
+Desrochers-2010-330-291	291	mondar	peel	PEEL	275	Verb	3.62745098039216	6	1	
+Desrochers-2010-330-292	292	mostrar	show	SHOW	1039	Verb	5.93	7	119	6.07
+Desrochers-2010-330-293	293	mutilar	mutilate			Verb	3.86274509803922	7	4	
+Desrochers-2010-330-294	294	nivelar	level			Verb	3.96078431372549	7	3	
+Desrochers-2010-330-295	295	nutrir	feed	FEED	739	Verb	3.84313725490196	6	5	
+Desrochers-2010-330-296	296	obsequiar	give away			Verb	3.93137254901961	9	2	
+Desrochers-2010-330-297	297	ofuscar	obfuscate			Verb	3.02941176470588	7		
+Desrochers-2010-330-298	298	ornar	decorate	DECORATE	1739	Verb	2	5	7	
+Desrochers-2010-330-299	299	pagar	pay	PAY	718	Verb	6.84	5	342	6.75
+Desrochers-2010-330-300	300	parar	stop	STOP DOING	3351	Verb	6.47058823529412	5	200	6.64
+Desrochers-2010-330-301	301	partir	share			Verb	5.88235294117647	6	79	5.78
+Desrochers-2010-330-302	302	pasar	go to			Verb	6.37254901960784	5	1040	6.33
+Desrochers-2010-330-303	303	pelar	peel	PEEL	275	Verb	6.04901960784314	5	12	
+Desrochers-2010-330-304	304	picar	click			Verb	5.92156862745098	5	14	
```

Comment:
picar means also "to itch"

Reply:
no, sorry, it means mainly "to sting" (mosquito)

Reply:
It can also be "to chop", but "to sting" or "to perforate" seems better indeed.